src/software/jetson_nano/rotary_encoder/RotaryEncoder.py

```diff
@@ -0,0 +1,123 @@
+import Jetson.GPIO as GPIO
+
+# Channel B and Channel A states
+STATE_0 = [0, 0]
+STATE_1 = [0, 1]
+STATE_2 = [1, 1]
+STATE_3 = [1, 0]
```

Comment:
A comment here explaining A/B states, and how it corresponds to the rotary encoder would be good.

Reply:
Added a comment to describe the states and how it corresponds to rotary encoder.